packages/site-kit/src/lib/components/Text.svelte

```diff
@@ -376,7 +380,7 @@
 			li::before {
 				content: '';
 				position: absolute;
-				top: 1.43rem;
+				top: 1.1rem;
```

Comment:
this looks bad on my screen. we will need to find some approach that works across different configurations
 
<img width="746" alt="image" src="https://github.com/user-attachments/assets/a1545bcd-4855-4522-af1d-a2fd4224b0d3">

Reply:
pushed something, let me know if that works for you

Reply:
saw you pushed something, how does that differ to the line-height dependent solution for you? can't we tweak the calc a bit? 
FWIW I'm fine with your solution, it's already much better than what we currently have

Reply:
It didn't work across the board for me, and I figured if everything is the same units we don't need a `calc`. Obviously `em` and `lh` are fungible, but the way I'm thinking of it is this: `lh` for spacing in typography, `em` for things that are tied to the size of the type itself (like icons), `rem` for layout. We're not being totally disciplined but I think that's a sensible way to think about things